test/service/test_version_service.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+
+from azul.logging import configure_test_logging
+from azul.version_service import (
+    VersionService,
+    VersionConflict,
+)
+from version_table_test_case import VersionTableTestCase
+
+
+def setUpModule():
```

Comment:
This should have a pragma suppressing the warning like so:

https://github.com/DataBiosphere/azul/commit/bccc31d29dfe9da710e0d1c0e4b03082de42886c